tasks/failover_ip.yml

```diff
@@ -0,0 +1,21 @@
+- name: Add failover network interface alias
+  template:
+    src=network/interfaces.d/failover-ip
```

Comment:
The YAML dictionary syntax is preferred these day, like you did further down for the `service` module.